lib/virt_autotest/utils.pm

```diff
@@ -246,4 +249,27 @@ sub add_guest_to_hosts {
     assert_script_run "echo '$address $hostname # virtualization' >> /etc/hosts";
 }
 
+# Remove additional disks from the given guest. We remove all disks that match the given pattern or 'vd[b-z]' if no pattern is given
+sub remove_additional_disks {
+    my $guest   = $_[0];
+    my $pattern = $_[1] // "vd[b-z]";
```

Comment:
I am afraid for different products( sle11/12/15), different hypervisors(xen/kvm), the disk name is not always vdxx?

Reply:
imho, there were always the different (virtual) guest disk (pattern) names for our SLE (11/12/15) product.

So, would be better to check with guest disk name firstly before remove additional disks.

Reply:
Thanks for the input. I share both of your observation here.
Do you perhaps have an overview of the different disk patterns, which we could use right away?

Otherwise I suggest we fix those issue as-they-appear because it's simply impossible to schedule verification runs for each product on time. I think by allowing the pattern as second optional argument we create the basis for this functionality and can include/extend the patterns then with time.

What do you think about this suggestion?

Reply:
@grisu48  sure, I do agree on this as your mentioned.

Quick idea, I had double confirmed this `$pattern` - guest disk name ( or Guest Target Name) on sles15sp3 xen host.
There were used with the had - FV (hvm) guest and xvda - PV guest as the default:
For example: 
```
virsh domblklist sles-15-sp3-64-fv-def-net
 Target   Source
-------------------------------------------------------------------
 hda      /var/lib/libvirt/images/sles-15-sp3-64-fv-def-net.qcow2

 virsh domblklist sles-15-sp3-64-pv-def-net
 Target   Source
-------------------------------------------------------------------
 xvda     /var/lib/libvirt/images/sles-15-sp3-64-pv-def-net.qcow2  
```
So, used with the current code `$pattern = $_[1] // "vd[b-z]";`, which will be not work on sles15sp3 xen host, imho.

Reply:
```suggestion
    my $pattern = $_[1] // "x?vd[b-z]";
```

Reply:
Good idea! Included it :+1: That should make sure that kvm+xen work. The current VR is a xen run.